akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -58,6 +59,12 @@ Scala
 Java
 :  @@snip [ClusterShardingTest.java]($code$/java/jdocs/sharding/ClusterShardingTest.java) { #counter-start }
 
+In some cases, the actor may need to know the `entityId` associated with it. This can be achieved in 2 ways:
+
+* Using the `propsFactory` parameter to `ClusterSharding.start`. The entity ID will be passed to the factory as a
+parameter, which can then be used in the creation of the actor (like the above example).
+* Using the actor's name (`self.path.name`). The actor's name will be the utf-8 URL-enconded entity ID.
```

Comment:
with this new api added I see no value in describing the alternative of using the self.path.name